### ISSUE TITLE
Edit the open-graph image in the SEO module

### ISSUE THREAD

**Issue by thasmo** [clickstorm/cs_seo#108]
Currently it's not possible to edit the open-graph image in the SEO module, which would be a feature saving lots of time compared to switching to individual pages. At the moment it only shows the value of the field but it's not possible to edit the image.

**Comment by mhirdes** [comment]
Yeah would be a great feature. But currently I don't know how to integrate the FAL in this view

**Comment by outdoorsman** [comment]
Yes, it would be nice to add a specific image that doesn't exist on the page... like on a homepage.

**Comment by mhirdes** [comment]
You can do this already in the page settings, but not in the module
